components/infobox/wikis/apexlegends/infobox_person_player_custom.lua

```diff
@@ -189,6 +190,14 @@ function CustomPlayer:defineCustomPageVariables(args)
 	else
 		Variables.varDefine('isplayer', 'true')
 	end
+	local lowercaseInput = _args.input and _args.input:lower() or nil
+	Variables.varDefine('input', _INPUTS[lowercaseInput] or _INPUTS['default'])
```

Comment:
```suggestion
```

Reply:
This was intended to be removed with the previous comment, I must have missed it. Apologies